Vagrantfile

```diff
@@ -6,11 +6,6 @@ Vagrant.configure("2") do |config|
     # Ubuntu 14.04 LTS
     config.vm.box = "ubuntu/trusty64"
 
-    # Ports to the services
```

Comment:
It is intended; once I removed `transit` and `chatta`, forwarding those ports became unnecessary.